src/controllers/delegates/controldelegate.cpp

```diff
@@ -48,7 +48,7 @@ QString ControlDelegate::displayText(const QVariant& value,
     }
 
     if (m_bIsIndexScript) {
-        return tr("Script: %1(%2)").arg(key.item, key.group);
+        return tr("%1(%2)").arg(key.item, key.group);
```

Comment:
```suggestion
        return tr("%1 (%2)").arg(key.item, key.group);
```
?

Looks like a function argument without the space.

Reply:
Actually it is.
But yes, it reads better with a space.

Reply:
> Actually it is.

Yes, but it's not the only one, so this is kind of misleading :D

Reply:
Actually, it's parenthesis + brackets what makes it look strange.
I'm not aware of any group that has no [ ]
So I'd remove the parenthesis, as well

Reply:
> So I'd remove the parenthesis, as well

ping. or should I merge?

Reply:
I'll finish it.

Reply:
No hurry, I just wondered because github notified me about "New changes since you last viewed" but nothing actually changed :D

Reply:
I experienced this earlier, too...dunno